### DISCUSSION TITLE
Fatal error

### DISCUSSION THREAD

**Discussion by pradeep10kumar** [UtrechtUniversity/yoda#314] [Q&A] [answered]
Hi,
When executing the line "ansible-playbook -i environments/development/allinone/ test.yml" from the URL https://utrechtuniversity.github.io/yoda/development/setting-up-development-environment.html, the following error is encountered: 

fatal: [combined -> 127.0.0.1]: FAILED! => {"changed": false, "dest": "./eus_index_page.tmp", "elapsed": 0, "msg": "Request failed", "response": "HTTP Error 404: Not Found", "status_code": 404, "url": "https://eus.yoda.test"}

PLAY RECAP ********************************************************************************************************************************************************************************************************************************************************************
combined                   : ok=96   changed=68   unreachable=0    failed=1    skipped=2    rescued=0    ignored=10

I have added following to host file:
192.168.56.10 portal.yoda.test
192.168.56.10 data.yoda.test
192.168.56.10 public.data.yoda.test
192.168.56.10 public.yoda.test
192.168.56.10 eus.yoda.test

**Comment by lwesterhof** [comment]
Hi,

Did the playbook finish without any errors? (`ansible-playbook -i environments/development/allinone/ playbook.yml`)

Which OS are you running?

**Reply by pradeep10kumar** [reply]
Thanks for quick your reply! I am running on Windows. The result of this command (ansible-playbook -i environments/development/allinone/ playbook.yml) is 

PLAY RECAP *****************************************************************************************************************************************************************
combined                   : ok=479  changed=14   unreachable=0    failed=0    skipped=231  rescued=0    ignored=1
localhost                  : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

Playbook run took 0 days, 0 hours, 8 minutes, 31 seconds

**Reply by lwesterhof** [reply]
Can you browse to https://eus.yoda.test ?

Should show this page:
![Screenshot 2023-10-25 at 11-38-50 Yoda External User Service ‐ Home](https://github.com/UtrechtUniversity/yoda/assets/29246264/a941f582-60f1-4999-9342-ab754db122db)

**Reply by pradeep10kumar** [reply]
Thank you once more! I am able to open this link in this browser with 
![image](https://github.com/UtrechtUniversity/yoda/assets/12733192/d5f223a9-c5ce-4cee-840b-f187dd9f8e4a)



I am sharing the errors I am encountering.

Error 1: TASK [yoda_test : Ensure Yoda licenses directory is present] ******************************************************************************************************************************************************************************************************************
fatal: [combined]: FAILED! => {"changed": true, "cmd": ["imkdir", "/tempZone/yoda/licenses"], "delta": "0:00:00.602542", "end": "2023-10-25 13:48:25.665781", "msg": "non-zero return code", "rc": 3, "start": "2023-10-25 13:48:25.063239", "stderr": "remote addresses: 127.0
.1.1 ERROR: mkdirUtil: mkColl of /tempZone/yoda/licenses error. status = -809000 CATALOG_ALREADY_HAS_ITEM_BY_THAT_NAME", "stderr_lines": ["remote addresses: 127.0.1.1 ERROR: mkdirUtil: mkColl of /tempZone/yoda/licenses error. status = -809000 CATALOG_ALREADY_HAS_ITEM_BY_
THAT_NAME"], "stdout": "", "stdout_lines": []}
...ignoringTASK [yoda_test_users_eus : Request EUS index page to trigger schema creation if needed] **************************************************************************************************************************************************************************************
fatal: [combined -> 127.0.0.1]: FAILED! => {"changed": false, "dest": "./eus_index_page.tmp", "elapsed": 0, "msg": "Request failed: <urlopen error [Errno -2] Name or service not known>", "url": "https://eus.yoda.test"}

PLAY RECAP ********************************************************************************************************************************************************************************************************************************************************************
combined                   : ok=96   changed=68   unreachable=0    failed=1    skipped=2    rescued=0    ignored=10

Error 2 : TASK [yoda_test_users_eus : Request EUS index page to trigger schema creation if needed] **************************************************************************************************************************************************************************************
fatal: [combined -> 127.0.0.1]: FAILED! => {"changed": false, "dest": "./eus_index_page.tmp", "elapsed": 0, "msg": "Request failed: <urlopen error [Errno -2] Name or service not known>", "url": "https://eus.yoda.test"}

PLAY RECAP ********************************************************************************************************************************************************************************************************************************************************************
combined                   : ok=96   changed=68   unreachable=0    failed=1    skipped=2    rescued=0    ignored=10

Playbook run took 0 days, 0 hours, 12 minutes, 8 seconds

**Reply by lwesterhof** [reply]
You have created the VMs with vagrant and are running Ansible on the controller VM?

Can you try adding `192.168.56.10 api.eus.yoda.test` to the /etc/hosts of the controller VM?

**Reply by pradeep10kumar** [reply]
I have added this to host. But still I am encountering the same issue:

![image](https://github.com/UtrechtUniversity/yoda/assets/12733192/ed12e133-1118-49b9-bc1f-d50fcddf31db)

Now I can not open "https://eus.yoda.test" in the Browser. 
"
This site can’t be reachedCheck if there is a typo in eus.yoda.test.
If spelling is correct, [try running Windows Network Diagnostics](javascript:diagnoseErrors()).
DNS_PROBE_FINISHED_NXDOMAIN
"

**Reply by lwesterhof** [reply, marked as answer]
> Can you try adding `192.168.56.10 api.eus.yoda.test` to the /etc/hosts of the controller VM?

Fixed in https://github.com/UtrechtUniversity/yoda/commit/01f3533af1e8d995b76c0b0bd3aab07e5b43c758

If you do a `git pull` and run `vagrant up --provision` it is fixed.
